### DISCUSSION TITLE
How i installed pantheon desktop ?

### DISCUSSION THREAD

**Discussion by TalhaAksoy** [elementary/triage#463] [Q&A]
i try pantheon desktop install debian but i cant pls help

**Comment by truefakts** [comment]
This is not an elementary OS question. But nonetheless, the answer to your question is as follows. Officially Debian supports the following desktop environments as presented in the installer:

- Debian desktop environment
- GNOME
- Xfce
- KDE Plasma
- Cinnamon
- MATE
- LXDE
- LXQt

As you can see, Pantheon is not in this list so you're kind of on your own. A quick web search presented this tutorial on a random blog, but your mileage may vary. https://linuxconfig.org/how-to-install-the-pantheon-desktop-on-debian

For best results, you may have better luck seeking guidance from the debian community directly

**Reply by TalhaAksoy** [reply]
I am trying to do a quick web search but I get this error 
```
 Wget: cannot resolve host address 'gandalfn.ovh'.

```

and i cant find any repo for downloading source code, u know working host address for pantheon desktop environment?
